src/client/java/teammates/client/scripts/RepairFeedbackSessionNameWithExtraWhiteSpace.java

```diff
@@ -0,0 +1,147 @@
+        migrator.doOperationRemotely();
+    }
+
+    @SuppressWarnings("deprecation")
+    @Override
+    protected void doOperation() {
+        Datastore.initialize();
+        
+        List<FeedbackSession> feedbackSessionList = feedbackSessionsDb.getAllFeedbackSessionEntities();
+        System.out.println("There is/are " + feedbackSessionList.size() + " session(s).");
+        
+        if (isPreview) {
+            System.out.println("Checking extra spaces in feedback session name...");
+        } else {
+            System.out.println("Removing extra spaces in feedback session name...");
+        }
+        
+        try {
+            int numberOfFeedbackSessionWithExtraWhiteSpacesInName = 0;
+            for (FeedbackSession session : feedbackSessionList) {
+                if (hasExtraSpaces(session.getFeedbackSessionName())) {
+                    numberOfFeedbackSessionWithExtraWhiteSpacesInName++;
+                    if (isPreview) {
+                        showFeedbackSession(session);
+                    } else {
+                        fixFeedbackSession(session);
+                    }
+                }
+            }
+            
+            if (isPreview) {
+                System.out.println("There are/is " + numberOfFeedbackSessionWithExtraWhiteSpacesInName
+                                   + "/" + feedbackSessionList.size() + " feedback session(s) with extra spaces in name!");
+            } else {
+                System.out.println("" + numberOfFeedbackSessionWithExtraWhiteSpacesInName
+                                   + "/" + feedbackSessionList.size() + " feedback session(s) have been fixed!");
+                System.out.println("Extra space removing done!");
+            }
+        } catch (InvalidParametersException | EntityDoesNotExistException | EntityAlreadyExistsException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    /**
+     * Displays the feedback session.
+     */
+    private void showFeedbackSession(FeedbackSession session) {
+        System.out.println("Feedback Session Name: \"" + session.getFeedbackSessionName() + "\" in " + session.getCourseId());
+    }
+
+    /**
+     * Remove extra spaces in feedback session's name
+     */
+    private void fixFeedbackSession(FeedbackSession session) throws InvalidParametersException, EntityDoesNotExistException, EntityAlreadyExistsException {
+        fixFeedbackQuestionsOfFeedbackSession(session);
+        fixFeedbackResponsesOfFeedbackSession(session);
+        fixFeedbackResponseCommentsOfFeedbackSession(session);
+        
+        FeedbackSessionAttributes sessionAttribute = new FeedbackSessionAttributes(session);
+        feedbackSessionsDb.deleteEntity(sessionAttribute);
+        sessionAttribute.feedbackSessionName = StringHelper.removeExtraSpace(sessionAttribute.getFeedbackSessionName());
+        feedbackSessionsDb.createEntity(sessionAttribute);
```

Comment:
I think as long as we change the feedback session name, the email links will not work anymore. I'm not sure what the best course of action here is too. Maybe we can run the script on preview first to see if this is actually a problem?